### DISCUSSION TITLE
Standings and Qualifying

### DISCUSSION THREAD

**Discussion by FearWhat** [MerlinCooper/iRacingReplayDirector#104] [Ideas]
Is there a way to configure this so it will do the Qualifying results and ending standings but have no overlays during the full highlight?  I am very sorry if i missed this function inside your tool already, I have been using this tool since this started. I love this tool Thank you for taking the time to keep this updated.

**Comment by MerlinCooper** [comment]
We’ll fastest should be to have a specific plugin. What overlay plugin are you using - might be able to try something this weekend

**Comment by MerlinCooper** [comment]
@FearWhat please let me know whether you are using Jocke or Standard Overlay. Will only do that for one of the two on short notice

**Reply by FearWhat** [reply]
Standard overlay , sorry I missed this email.

**Reply by FearWhat** [reply]
Thank you 😊,  

https://youtu.be/PDnP_g9Wnvk  

I don’t know if the link will work.  I really forgot to add the link to your tool.
